### ISSUE TITLE
Penile pain

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10017]
**Preferred term label:**
Penile pain

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Penis fracture

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000088
- label: Penile pain
- parents: [Pain (HP:0012531)](https://hpo.jax.org/app/browse/term/HP:0012531)
- pmid: 
- def: An unpleasant sensation characterized by physical discomfort (such as pricking, throbbing, or aching) and perceived to originate in the penis.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10017
- synonyms:

**Comment by MickeySegal** [comment]
This seems like a duplicate of https://github.com/obophenotype/human-phenotype-ontology/issues/10099

**Comment by pnrobinson** [comment]
yes, this was a duplicate, I will fix, reopening, thanks for picking this up

**Comment by pnrobinson** [comment]
Thanks, I removed this. The correct id is
HP:0040411 Pain in penis, synonym Penile pain
